gitpod-io/gitpod
# [installer] Ensure stable ordering of rendered K8s objects

## Description
This PR ensures that the ordering of the rendered K8s objects is stable. That allows comparing renderings with different configs.


## How to test

Render with 2 different configs (e.g. with external registry and with internal registry). Without this change, the ordering is not stable and it is impossible to compare the changes. After this change, the ordering is stable and you can see with a diff what has changed in the rendering.

#11880 has an example where you can compare the golden files with and without an external registry.

## Release Notes

```release-note
NONE
```

## Werft options:
